components/FlexibleHeader/src/MDCFlexibleHeaderView+ShiftBehavior.h

```diff
@@ -129,6 +129,9 @@ typedef NS_ENUM(NSInteger, MDCFlexibleHeaderContentImportance) {
 /** Asks the receiver to take the header off-screen if it's currently on-screen. */
 - (void)shiftHeaderOffScreenAnimated:(BOOL)animated;
 
+/** Indicates whether the header is presently shifted offscreen. */
```

Comment:
At what point does it shift from "No" to "yes" and back? Is it if any single pixel of the header is "off-screen" or if all of it is off screen or some significant portion (10%)?

Reply:
Clarified the docs.